src/mixins/filing-template-mixin.ts

```diff
@@ -107,7 +108,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
     let filing: CorrectionFilingIF = {
       header: {
         name: FilingTypes.CORRECTION,
-        certifiedBy: this.getCertifyState.certifiedBy,
+        certifiedBy: this.getCertifyState.certifiedBy || '',
```

Comment:
As per [schema](https://github.com/bcgov/business-schemas/blob/82a79a365b288ccd502e7299728977908311c473/src/registry_schemas/schemas/filing.json#L87), this is a required string, so it cannot be undefined or null.